src/Sylius/Bundle/ShopBundle/Resources/views/Common/Form/_address.html.twig

```diff
@@ -7,7 +7,7 @@
 
 {% include '@SyliusShop/Common/Form/_countryCode.html.twig' with {'form': form.countryCode} %}
 
-<div class="province-container field" data-url="{{ path('sylius_shop_ajax_render_province_form') }}">
+<div class="province-container required field" data-url="{{ path('sylius_shop_ajax_render_province_form') }}">
```

Comment:
Can we make it dependent on the `form.provinceCode`'s / `form.provinceName`'s `required` option?

Reply:
I agree that's a better solution, my solution can probably be applied as a template override to fix it temporarily, thanks @pamil .

Reply:
I've made a bit of research and it's required only when `form.provinceCode` is defined.